recipes/pint-pulsar/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "pint-pulsar" %}
+{% set version = "0.6.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3ab01cb3d7f6ddbad9fd62a7e7f4049105fbffa326d193d7fed1ba3e66408dae
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: true  # [py27]
+  skip: true  # [win]
+# Add [skip appveyor] to commit messages to prevent windows from starting at all
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+
+  run:
+    - python
+    - numpy >=1.16
+    - astropy >=3.2,<=4.0
+    - scipy >=0.18.1
+    - jplephem >=2.6
+    - matplotlib >=1.5.3
```

https://github.com/conda-forge/conda-forge.github.io/issues/933
```suggestion
    - matplotlib-base >=1.5.3
```